browser/src/Config.ts

```diff
@@ -80,8 +80,8 @@ const DefaultPlatformConfig = Platform.isWindows() ? WindowsConfig : Platform.is
 
 Performance.mark("Config.load.start")
 
-const userConfigFile = path.join(getUserFolder(), "config.json")
-const userJsConfig = path.join(getUserFolder(), "config.js")
+export const userConfigFile = path.join(getUserFolder(), "config.json")
+export const userJsConfig = path.join(getUserFolder(), "config.js")
```

Comment:
Not confused, totally valid point. `config.json` was added first, but `config.js` is like configuration v2... There were some things for my config that I wanted to use JavaScript for (for example, have a random background image on boot). We should remove `config.json`. It would be great to track in a separate issue and make sure that the documentation is up-to-date (there might still be references to `config.js`).